egs/fisher_swbd/s5/local/chain/run_tdnn_lstm_1a_svd.sh

```diff
@@ -0,0 +1,363 @@
+#!/bin/bash
+# same as run_tdnn_opgru_1a.sh, but replacing Norm-OPGRU with LSTMP.
+# ./local/chain/compare_wer_general.sh --looped tdnn_lstm_1a_sp
+# System                tdnn_lstm_1a_sp
+# WER on eval2000(tg)        12.3
+#           [looped:]        12.2
+# WER on eval2000(fg)        12.1
+#           [looped:]        12.1
+# WER on eval2000(fg)
+#  [SVD retrained + looped]  12.1
+# WER on rt03(tg)            11.6
+#           [looped:]        11.6
+# WER on rt03(fg)            11.3
+#           [looped:]        11.3
```

Comment:
where is the rt03 SVD-retrained number?

Reply:
Need to put those in. Didn't calculate I believe yet.

Reply:
Using the above SVD retrained 48Mb model, we get 12% WER on the entire rt03 dataset, using the multi-en trigram LM trained by adding Fisher and swbd texts.
On swbd part of rt03 : WER 14.1% (tg)
On Fisher part of rt03 : WER 9.7% (tg)